Plugins/Public/base_plugin/PlayerCommands.cpp

```diff
@@ -1781,6 +1783,32 @@ namespace PlayerCommands
 				if (curr_item == item)
 				{
 					i->second.price = (float)money;
+					SendMarketGoodUpdated(base, i->first, i->second);
+					base->Save();

```

Comment:
This is more for something else but worth mentioning. Base saving should be delegated to threads and done periodically over time.